src/Infrastructure/BotSharp.OpenAPI/Controllers/KnowledgeController.cs

```diff
@@ -27,44 +28,18 @@ public async Task<List<RetrievedResult>> RetrieveKnowledge([FromRoute] string ag
     }
 
     [HttpPost("/knowledge/{agentId}")]
-    public async Task<IActionResult> FeedKnowledge([FromRoute] string agentId, List<IFormFile> files, [FromQuery] int? startPageNum, [FromQuery] int? endPageNum)
+    public async Task<IActionResult> FeedKnowledge([FromRoute] string agentId, List<IFormFile> files, [FromQuery] int? startPageNum, [FromQuery] int? endPageNum, [FromQuery] bool? paddleModel)
     {
         long size = files.Sum(f => f.Length);
 
         foreach (var formFile in files)
         {
-            if (formFile.Length <= 0)
-            {
-                continue;
-            }
-
-            var filePath = Path.GetTempFileName();
-
-            using (var stream = System.IO.File.Create(filePath))
-            {
-                await formFile.CopyToAsync(stream);
-            }
-
-            var document = PdfDocument.Open(filePath);
```

Comment:
We can keep `PdfPig` as one of the implementation of `IPdf2TextConverter` rather than delete it.